_posts/2017-06-23-announcing-angular-master-class-in-houston.md

```diff
@@ -12,11 +12,6 @@ tags:
   - announcements
 author: christoph_burgdorf
 related_posts:
-  - Announcing Angular Master Class in Berlin
-  - Join our free meetup in Denmark
-  - Announcing Angular Master Class in Denmark
-  - Dominic joins thoughtram
-  - Announcing Angular Master Class in Freiburg
```

Comment:
Oh non of them. That's the thing. You removed them already in your commit, but they will be re-added automatically.